plugins/continuous_dataset.cc

```diff
@@ -518,13 +518,25 @@ reconstituteConfig(const MatrixNamedRow & row)
 {
     Json::Value current;
 
+    auto pruneDoubleQuotes = [] (const string & in) {
```

Comment:
parse_path() will take a string and make a path
path_element() will take a path and give you a part of it